### PR TITLE
Added USAGE.md and made corrections in README.md

### LINKED ISSUE #108: Detailed documentation is unavailable for usage of kolibri2zim
Current state of repository does not contain any user instruction documents for beginners, on how to make use of kolibri2zim tool. It would be really helpful for the new users if there is a user instruction manual.

### LINKED ISSUE #115: Incorrect Docker image name README.md
Description:
Docker image name README.md of new_ui branch is incorrect.

### PR DESCRIPTION
**Pull request by Sudarshan-21**
resolves #108, resolves #115
Corrections made:
USAGE.md added for usage of kolibri2zim using docker and corrected docker image name in README.md

### CODE REVIEW

**Comment by benoit74** [comment]
Is this ready for review?

**Comment by Sudarshan-21** [comment]
> Is this ready for review?

yes it is.

**Review by benoit74 — CHANGES_REQUESTED** [review]
Thank you! Few things to be addressed before merge.

Please simplify PR title (it is way too long) and move details to the first comment
Please use a proper link to issue in first comment so that the issue will automatically be closed when PR is merged (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
Please address other comments (or speak up)

**Comment by Sudarshan-21** [comment]
@benoit74 I mde the instructed changes, let me know if there is anything more.

File: USAGE.md [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
 
 ```bash
 docker run -v my_dir:/output ghcr.io/openzim/kolibri kolibri2zim --channel-id `channel-id` --name `name-of-the-channel`
 ```
 
-- This will create a `.zim` file in the `/output` file in the docker container.
+- This will create a `.zim` file in the `/output` file, which will be persisted in the my_dir Docker volume.
 
--For getting this .zim file on to your local machine you can save it to your desktop by using `save` option from docker.
+-For getting this .zim file on to your local machine you can save it to your desktop by using `save` command(right click on mouse).
```
Comment (benoit74) [inline USAGE.md:15]: missing space: `command(right click on mouse).` => `command (right click on mouse).`

**Comment by Sudarshan-21** [comment]
@benoit74 did those minor changes, please check.

**Review by benoit74 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**